README.md

```diff
@@ -59,7 +59,7 @@ The table below shows the track parameters that can be adjusted along with defau
 | Parameter | Explanation | Type | Default Value |
 | --------- | ----------- | ---- | ------------- |
 | `number_of_replicas` | Number of index replicas | `int` | `0` |
-| `shard_count` | Number of primary shards | `int` | `2` |
+| `number_of_shards` | Number of primary shards | `int` | `2` |
```

Comment:
This is obsolete here in the sense that this challenge does not set a different number of shards (the default in the index template is two shards already). I think we can safely remove the whole line here and instead document this as a track parameter available for all challenges? This applies similarly to `number_of_replicas`. However, we can merge this PR as is and do the cleanup in a followup PR.